PharoTour/PharoTour.pillar

```diff
@@ -334,8 +333,8 @@ it"".
 
 +Inspecting an object>file://figures/inspector.png|width=80|label=fig:inspector+
 
-Close the inspector. Type the expression ==Object== in a playground and this
-time ""explore it"" (==CMD-I==, that is, the uppercase i, or ==CMD-SHIFT-i==).
+Close the inspector. Type the expression [[[language=smalltalk Object]]]
```

Comment:
please keep using == when writing inline code and [[[ for code blocks. Using [[[ in the middle of a line will not be possible in the next version of Pillar.